src/nvim/buffer.c

```diff
@@ -5084,17 +5083,16 @@ static int sign_compare(const void *a1, const void *a2)
     if (s1->id < s2->id) {
         return -1;
     }
-
     return 0;
 }
 
-int buf_signcols(buf_T *buf)
+int buf_signcols(buf_T *buf, int max_signs)
```

Comment:
better s/max_signs/max_signs_per_line/ probably.

Reply:
Don't need the long name. Sign width is by definition per-line.